modules/machine-user-infra-machines-iso.adoc

```diff
@@ -2,19 +2,46 @@
 //
 // * machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc
 // * post_installation_configuration/node-tasks.adoc
+ifeval::["{context}" == "multi-architecture-configuration"]
+:multi:
+endif::[]
 
 :_content-type: PROCEDURE
 [id="machine-user-infra-machines-iso_{context}"]
-= Creating more {op-system} machines using an ISO image
+= Creating {op-system} machines using an ISO image
 
 You can create more {op-system-first} compute machines for your bare metal cluster by using an ISO image to create the machines.
 
 .Prerequisites
 
 * Obtain the URL of the Ignition config file for the compute machines for your cluster. You uploaded this file to your HTTP server during installation.
+* You installed the OpenShift CLI (`oc`)
 
 .Procedure
 
+. You can extract the Ignition config file from the cluster by running the following command:
```

Comment:
Is this something the user needs to do to go through the steps following? If so, it might be best to remove "can" because the way it is worded it almost feels like its an optional step. You could reword to something like:
```suggestion
. Extract the Ignition config file from the cluster by running the following command:
```